packages/starlight/utils/translations.ts

```diff
@@ -8,6 +8,7 @@ let userTranslations: Record<string, i18nSchemaOutput> = {};
 try {
 	// Load the user’s i18n collection and ignore the error if it doesn’t exist.
 	userTranslations = Object.fromEntries(
+		// @ts-expect-error
```

Comment:
I think it might be safer to ignore this one — this is only an error if a user doesn’t have an i18n collection:
```suggestion
		// @ts-ignore — may be an error in projects without an i18n collection

```

Reply:
Worth checking whether that leaks into starlight. It might need to become a js file instead. I'll take a look.